source/beamline/CLS/CLSJJSlitBladesControl.h

```diff
@@ -3,66 +3,53 @@
 
 #include <QSignalMapper>
 
-#include "beamline/AMCompositeControl.h"
+#include "beamline/AMPseudoMotorControl.h"
 #include "actions3/AMAction3.h"
 
 // Values.
 
-#define CLSJJSLITSCONTROL_VALUE_MIN -30
-#define CLSJJSLITSCONTROL_VALUE_MAX 30
+#define CLSJJSLITBLADESCONTROL_VALUE_MIN -30
```

Comment:
Are these actually hard coded values?  Are they different from using your setMinimumValue and setMaximumValue?